Source/Renderer/Context.js

```diff
@@ -278,13 +293,39 @@ define([
         this._textureFilterAnisotropic = textureFilterAnisotropic;
         ContextLimits._maximumTextureFilterAnisotropy = defined(textureFilterAnisotropic) ? gl.getParameter(textureFilterAnisotropic.MAX_TEXTURE_MAX_ANISOTROPY_EXT) : 1.0;
 
-        this._vertexArrayObject = getExtension(gl, ['OES_vertex_array_object']);
+        var vertexArrayObject = getExtension(gl, ['OES_vertex_array_object']);
+        this._vertexArrayObject = vertexArrayObject;
+        this.glCreateVertexArray = function() {
```

Comment:
Throughout, why not:
```javascript
this.glCreateVertexArray = webgl2 ? gl.createVertexArray : vertexArrayObject.createVertexArrayOES;
```
This will be fast than using the closure.

Also, I don't know that `Context` is the best place to stash these function pointers.  I'll see if I have any ideas as I continue to review this.